static/templates/delete_topic_modal.hbs

```diff
@@ -13,7 +13,7 @@
     <div class="modal-footer">
         <button class="button" data-dismiss="modal">{{t "Cancel" }}</button>
         <button class="button btn-danger rounded" id="do_delete_topic_button">
-            {{t "Delete topic" }}
+            {{t "Confirm" }}
```

Comment:
<img width="427" alt="delete_topic_confirm_modal" src="https://user-images.githubusercontent.com/59444243/113509239-1cfbb380-9572-11eb-8c09-7fd294124799.PNG">

Reply:
Let's move the question to the bottom here.